docs: Update meta content - Concepts (#5224)
hashicorp/boundary: website/content/docs/concepts/domain-model/credentials.mdx

---
layout: docs
page_title: Domain model - credentials
description: >-

>   Use credentials to define a user's permissions on a host. Learn about username password, SSH private key, SSH certificate, and JSON credential types.

Reviewer: ```suggestion
  Use credentials to define secrets for a host. Learn about username password, SSH private key, SSH certificate, and JSON credential types.
```
